marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/impl/security/getAuthoritiesTest.sjs

```diff
@@ -45,6 +45,15 @@ hubTest.runWithRolesAndPrivileges(['hub-central-load-reader'], [], function() {
     assertions.push(test.assertFalse(authorities.includes('writeIngestion'), 'hub-central-load-reader should not have "writeIngestion"'));
 });
 
+// Test hub-central-load-writer
+hubTest.runWithRolesAndPrivileges(['hub-central-load-writer'], [], function() {
```

Comment:
Are these assertions necessary since you have the tests on the controller, which verifies that the ROLE_* stuff works, which means the right authorities are being returned?

Reply:
Will the controller test fail, if the role doesnt include loginToHubCentral or readIndestion authorities?

Reply:
Yes - the loginAsTestUser method in AbstractMvcTest is making a POST to /api/login, so it's going through all the same plumbing as when a user authenticates in HC. So if a call is made with a user that can't login, the test will immediately fail.

Reply:
OK Thanks. I will update the test plan likewise.